cdc/processor/processor.go

```diff
@@ -981,29 +983,17 @@ func (p *processor) createTablePipelineImpl(ctx cdcContext.Context, tableID mode
 
 	sink := p.sinkManager.CreateTableSink(tableID, replicaInfo.StartTs, p.redoManager)
 	var table tablepipeline.TablePipeline
-	if config.GetGlobalServerConfig().Debug.EnableTableActor {
```

Comment:
Could you also add some comments to EnableTableActor in config/debug.go that it has been deprecated.